pkg/cluster/context.go

```diff
@@ -181,7 +183,7 @@ func (c *Context) provisionControlPlane(
 	if err := node.FixMounts(); err != nil {
 		// TODO(bentheelder): logging here
 		// TODO(bentheelder): add a flag to retain the broken nodes for debugging
-		nodes.Delete(node)
+		nodes.Delete(*node)
```

Comment:
@munnerz the pointers came back to bite me, so this is my compromise for now

Reply:
Perhaps we change it to `[]nodes.Node` soon, and use `&node` where pointers are required 🙄

Reply:
yeah, that bit me :^)